src/main/zapHomeFiles/hud/utils.js

```diff
@@ -497,7 +497,16 @@ var utils = (function() {
 			})
 			.catch(errorHandler);
 	}
-	
+
+	function zapApiErrorDialog(tabId,  response) {
```

Comment:
is there a place where we'll use the error dialog vs just logging the error? or should we put it all failure cases? curious if you had thought about that or if you justed wanted to surface these error messages.

Reply:
I think it all depends on the caller - this is a log level function so I dont think it should dictate how the failure is handled